contracts/examples/Collateralized.sol

```diff
@@ -0,0 +1,126 @@
+/*
+
+  Copyright 2017 Dharma Labs Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity 0.4.18;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+
+
+contract Collateralized {
+    using SafeMath for uint;
+
+    struct Collateral {
+        address user;
+        address token;
+        uint amount;
+        uint lockupPeriod;
+    }
+
+    address public beneficiary;
```

Comment:
Same comment as above -- w/ the `issuanceCommitmentHash`, you can retrieve the `beneficiary` from the `DebtRegistry`